pepper-apis/src/main/java/org/broadinstitute/ddp/service/actvityinstancebuilder/block/question/QuestionUtil.java

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.ddp.service.actvityinstancebuilder.block.question;
+
+import org.broadinstitute.ddp.model.activity.definition.question.QuestionDef;
+import org.broadinstitute.ddp.service.actvityinstancebuilder.ActivityInstanceFromActivityDefStoreBuilder;
+
+public class QuestionUtil {
+
+    public static boolean isReadOnly(ActivityInstanceFromActivityDefStoreBuilder.Context context, QuestionDef questionDef) {
```

Comment:
This is not feature-parity with existing calculation of question read-only. Please see `ddp.util.QuestionUtil` class. Also, I recommend removing this class here and add a new method in the other util class -- having two classes with same name is a bit confusing.
```
isReadOnly(Handle handle, QuestionDef questionDef, InstanceStatusType instanceStatus, long instanceId)
```